### PR TITLE
add new figure

### PR DESCRIPTION
**Pull request by tomoya-sasaki**
A figure inspired by `plot(., type="summary")` in stm

### CODE REVIEW

**Comment by Shusei-E** [comment]
@tomoya-sasaki Please check my edits!

**Comment by tomoya-sasaki** [comment]
Thank you! This looks great to me.

File: R/plots.R [hunk 1]
```diff
@@ -233,6 +234,7 @@ plot_pi <- function(x, show_topic = NULL, start = 0, ci = 0.9, method = c("hdi",
 #'
 #' @param x the output from a keyATM model (see [keyATM()]).
 #' @param n The number of top words to show. Default is \code{3}.
+#' @param var_name the name of the variable in the plot.
```
Comment (Shusei-E) [inline R/plots.R]: @tomoya-sasaki Is this topic name?

**Comment by tomoya-sasaki** [comment]
Can I merge this to `package_v4dev`? Or do we need any additional edits related to this?

**Comment by Shusei-E** [comment]
@tomoya-sasaki Yes, please!

**Pull request merged**